### ISSUE TITLE
Add unambiguous links to versions in column selected

### ISSUE THREAD

**Issue by simon04** [repology/repology-webapp#118]
https://repology.org/projects/?inrepo=arch

The column selected is unambiguously identifying Arch packages. Thus, link the versions to the corresponding packages on archlinux.org

**Comment by AMDmi3** [comment]
Makes sense, however it conflicts with #88 and my plans for a database refactoring which implies making less (unused) data to be fetched in order to render projects list, which would make links unavailable there. Need to think more on this.
